### PR TITLE
feat: loopserver chart

### PR DESCRIPTION
**Pull request by ntheile**
This PR adds a new chart called loopserver to be used in dev charts (regtest) and smoketest environments. It also copies the loop macaroon to kubernetes secrets to be consumed by the galoy main backend and cron servers.

### CODE REVIEW

**Comment by ntheile** [comment]
@openoms @bodymindarts ,

Could you test the `dev charts` locally in your environments? I added a `loopserver` helm chart that runs if you are in a regtest environment. I also was able to copy the `loop macaroons` successfully to `kubernetes secrets`.

You should be able to test as follows:

Boot dev charts
```
direnv allow
make create-cluster
make init
make deploy-services
make deploy
```

You should see the pods `lnd1-loop-0` and `lnd1-loopserver-0`
```
kubectl get pods -A | grep loop
```

Next you should see the loop macaroons copied over to secrets
```
kubectl -n galoy-dev-bitcoin describe secret lnd1-loop-credentials
```

Lastly you should be able to test `loop quote` talking to a local regtest loopserver
```
kubectl -n galoy-dev-bitcoin exec lnd1-loop-0 -- loop -n regtest quote out 250000
```

**Comment by openoms** [comment]
Looking good:
```
$ k get pods -A | grep loop
galoy-dev-bitcoin      lnd1-loopserver-0                                          1/1     Running     0          10m
galoy-dev-bitcoin      lnd1-loop-0                                                2/2     Running     0          10m
```

```
$ kubectl -n galoy-dev-bitcoin describe secret lnd1-loop-credentials
Name:         lnd1-loop-credentials
Namespace:    galoy-dev-bitcoin
Labels:       <none>
Annotations:  <none>

Type:  Opaque

Data
====
loop.macaroon:         164 bytes
loop_macaroon_base64:  220 bytes
tls.cert:              826 bytes
tls_base64:            1104 bytes
```

```
$ kubectl -n galoy-dev-bitcoin exec lnd1-loop-0 -- loop -n regtest quote out 250000
Defaulted container "loop" out of: loop, export-secrets
Send off-chain:                            250000 sat
Receive on-chain:                          240788 sat
Estimated total fee:                         9212 sat
```

The only nit I found is to be consistent with the versions:
loop v0.20.1 in the Dockerfile
and 
loopserver v0.9.52 in the Chart.yaml Appversion, chart version: v0.1.0
but it is fully functional anyway!

**Comment by ntheile** [comment]
> The only nit I found is to be consistent with the versions:
> loop v0.20.1 in the Dockerfile
> and
> loopserver v0.9.52 in the Chart.yaml Appversion

Yeah, latest version of the loopserver (v0.9.57-beta) doesn't work at all for me. Even last month I could not get version (v0.9.54-beta) working

File: images/lnd-sidecar/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
 FROM lightninglabs/lnd:v0.15.0-beta as lnd
+FROM lightninglabs/loop:v0.19.1-beta as loop
```
Comment (openoms) [inline images/lnd-sidecar/Dockerfile:2]:
```suggestion
FROM lightninglabs/loop:v0.20.1-beta as loop
```

File: charts/lnd/charts/loopserver/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.4
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: 0.11.1
```
Comment (openoms) [inline charts/lnd/charts/loopserver/Chart.yaml:24]:
```suggestion
appVersion: v0.9.52
```

File: charts/lnd/charts/loopserver/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.4
```
Comment (openoms) [inline charts/lnd/charts/loopserver/Chart.yaml:18]:
```suggestion
version: 0.1.0
```

**Review by openoms — COMMENTED** [review]
added suggestions on what I meant

File: images/lnd-sidecar/Dockerfile (hunk 1 above)
Comment (ntheile) [reply images/lnd-sidecar/Dockerfile:2]: oh yes! thanks for catching that

**Review by vindard — APPROVED** [review]
Approving based on @openoms reviews and latest comments, and fix commits after those comments

**Pull request merged**